packages/management/framework/bundle/config/pacakgeinstalls/addons-manager-service-account.yaml

```diff
@@ -0,0 +1,37 @@
+#@ load("@ytt:data", "data")
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tanzu-addons-manager-package-sa
+  namespace: #@ data.values.namespaceForPackageInstallation
+  annotations:
+    kapp.k14s.io/change-group: "addons-manager-packageinstall/serviceaccount-0"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: tanzu-addons-manager-package-cluster-role
+  annotations:
+    kapp.k14s.io/change-group: "addons-manager-packageinstall/serviceaccount-0"
+rules:
+- apiGroups: ["*"]
+  resources: ["*"]
+  verbs: ["*"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: tanzu-addons-manager-package-cluster-rolebinding
+  annotations:
+    kapp.k14s.io/change-rule.0: "upsert after upserting addons-manager-packageinstall/serviceaccount-0"
```

Comment:
You added the below change rules to PackageInstall and Secret
```
    kapp.k14s.io/change-rule.0: "upsert after upserting featuregates-packageinstall/serviceaccount"
    kapp.k14s.io/change-rule.1: "delete before deleting featuregates-packageinstall/serviceaccount"
```
but you missed adding the `change-group` for those change rules

This below change-group needs to be added here
```
kapp.k14s.io/change-group: "addons-manager-packageinstall/serviceaccount"
```

Reply:
Nice catch. Thanks!